src/components/SearchContainer/SearchContainer.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import ToggleComponent from '../ToggleComponent/ToggleComponent';
+import SearchBar from '../SearchBar/SearchBar';
+import './SearchContainer.scss';
+
+const SearchOptions = {
```

Comment:
If it uses more than one time it needs to be a constant, moved to constants folder and import where it used